scripts/zones/Crawlers_Nest_[S]/mobs/Morille_Mortelle.lua

```diff
@@ -26,3 +38,26 @@ function onMobDespawn(mob)
     DeterMob(PH, false);
     GetMobByID(PH):setRespawnTime(GetMobRespawnTime(PH));
 end;
+
+-----------------------------------
+-- onAdditionalEffect
+-----------------------------------
+
+function onAdditionalEffect(mob, player)
+    local chance = 25;
+    local resist = applyResistanceAddEffect(mob,player,ELE_WATER,EFFECT_PLAGUE);
+    if (math.random(0,99) >= chance or resist <= 0.5) then
+        return 0,0,0;
+    else
+        local duration = 30;
+        if (mob:getMainLvl() > player:getMainLvl()) then
+            duration = duration + (mob:getMainLvl() - player:getMainLvl())
+        end
+        duration = utils.clamp(duration,1,45);
+        duration = duration * resist;
+        if (not player:hasStatusEffect(EFFECT_PLAGUE)) then
```

Comment:
See: https://github.com/DarkstarProject/darkstar/pull/3917#discussion_r123145549

I will wait to see if any other project members comment for a few days.